### PR TITLE
Update Pages “demo_friday”

### PR DESCRIPTION
**Pull request by kristinedill**
Automatically generated by Netlify CMS

### CODE REVIEW

**Review by kylekirkby — APPROVED** [review]

**Comment by kristinedill** [comment]
hi @kylekirkby is it possible that when I click deploy preview it takes me to the page i just edited? This takes me to the homepage, but i cant find the demo page to preview it because its not in the nav bar. 
So when this happens, I have to add it to the nav bar and publish and then come back here and preview it again. 

Thanks!

**Comment by kylekirkby** [comment]
> hi @kylekirkby is it possible that when I click deploy preview it takes me to the page i just edited? This takes me to the homepage, but i cant find the demo page to preview it because its not in the nav bar.
> So when this happens, I have to add it to the nav bar and publish and then come back here and preview it again.
> 
> Thanks!

Hi @kristinedill, I don't think that would be reasonably possible directly from GitHub.

I've just created a PR to enable "Deploy Preview" links directly within NetlifyCMS. We have this enabled for the Linaro.org CMS so I'm not sure why it's not enabled for connect.linaro.org.  When this has been pushed through, you'll see a "View Preview" button once the site has been built and ready to be viewed. See https://linaro.atlassian.net/wiki/spaces/IS/pages/4102063136/Saving+And+Publishing+Changes for more info.

You can always manually type in the URL of the page you're editing/creating in the address bar in your browser.

**Comment by kristinedill** [comment]
@kylekirkby I tried to change the name of this page in Netlify to demo-showcase because it's not on Friday this time.

Is it possible to change this? I see it's back to demo-friday 

Thanks, 

Kristine

File: _pages/demo_friday.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 title: Call for Demos
-description: >
-  At every Linaro Connect we end the week on a high note with our Demo Friday
-  Technology Showcase.
-permalink: /demo-friday/
+description: Submit your application to participate in the Virtual Demo
+  Technology Showcase on Thursday, September 9.
+permalink: /demo-showcase/
```
Comment (kylekirkby) [inline _pages/demo_friday.md:5]: @kristinedill this is still /demo-showcase/?

**Comment by kylekirkby** [comment]
> @kylekirkby I tried to change the name of this page in Netlify to demo-showcase because it's not on Friday this time.
> 
> Is it possible to change this? I see it's back to demo-friday
> 
> Thanks,
> 
> Kristine

https://production-connect-linaro-org-828.ghactions.linaro.org/demo-showcase/

**Pull request merged**